### ISSUE TITLE
Script always erases disk4 on macOS

### ISSUE THREAD

**Issue by corpnewt** [notthebee/macos_usb#1]
Per [this line](https://github.com/notthebee/macos_usb/blob/master/macos_usb.sh#L98), the script always erases disk4 regardless of what the user selects.
```sh
sudo diskutil eraseDisk JHFS+ INSTALL /dev/disk4
```
Changing it to the following should preserve the user's selection:
```sh
sudo diskutil eraseDisk JHFS+ INSTALL ${flashdrive}
```

-CorpNewt

**Comment by notthebee** [comment]
Not sure how I missed that! 😁 Thanks
Fixed in d3533a007347865df3b9fc5f4c8fa61a1c496120

**Comment by corpnewt** [comment]
Glad to hear that fixed it.

Happy hacking,

-CorpNewt
